test/e2e/apimachinery/discovery.go

```diff
@@ -155,4 +157,118 @@ var _ = SIGDescribe("Discovery", func() {
 			framework.ExpectEqual(true, match, "failed to find a valid version for PreferredVersion")
 		}
 	})
+
+	ginkgo.It("should locate the groupVersion and a resource within each APIResource", func(ctx context.Context) {
+
+		tests := []struct {
+			apiBasePath   string
+			apiResource   string
+			apiVersion    string
+			groupVersion  string
```

Comment:
no need to add redundant `groupVersion` info into the table test, construct it inside the test from `(schema.GroupVersion{Group: t.apiGroup, Version: t.apiVersion}).String()`

Reply:
Fixed